dbt_style_guide.md

```diff
@@ -233,5 +355,37 @@ models:
 
 ## Jinja style guide

```

Comment:
Should we mention that `{%-` and `-%}` can be used to [control whitespace](https://docs.getdbt.com/guides/getting-started/learning-more/using-jinja#use-whitespace-control-to-tidy-up-compiled-code)?

Reply:
Voting 👍 on this

I would love it if it said something like “Use [Jinja’s whitespace control](https://jinja.palletsprojects.com/en/3.1.x/templates/#whitespace-control) so that rendered SQL can be easily read.” 

I almost feel this is too prescriptive without saying “don’t spend too much time on this”, haha.

This is the part I’m struggling with - 
I want something that’s not _too_ prescriptive but also I feel any good macro is using _some_ whitespace control. 😅

Reply:
I feel like "use whitespace control" with the caveat on not trying to make it perfect is the right balance.

Reply:
Done - [Commit](https://github.com/dbt-labs/corp/pull/63/commits/c1c88f55fa6a80ca8103325d1de4654ae3941258)

As far as how to whitespace control for for loops, etc, I think we should be careful and ensure we're not making this into a "how to" guide. There are multiple ways you could control the whitespace when making a `for` loop and everyone will do it different (for example, I tend to put {%- if not loop.last %},{% endif %} and others put newlines on the block and add extra whitespace controls). Going to leave this suggestion out.